astropy/coordinates/representation/base.py

```diff
@@ -145,13 +145,14 @@ class BaseRepresentationOrDifferential(MaskableShapedLikeNDArray):
 
     # Ensure multiplication/division with ndarray or Quantity doesn't lead to
     # object arrays.
-    __array_priority__: ClassVar[int] = 50000
+    __array_priority__: Final[int] = 50000
```

Comment:
I think one can remove `int` too, since that is obvious (sorry, super nitpicky, but I think new `ruff` actually triggers on this).

Reply:
SGTM.

Reply:
Done.